power_of_n.py

```diff
@@ -0,0 +1,57 @@
+#Python program to calculate x raised to the power n (i.e., x^n)
+
+# Script Name		: power_of_n.py
+# Author		    : Himanshu Gupta
+# Created			: 2nd September 2023
+# Last Modified		:
+# Version			: 1.0
+# Modifications		:
+# Description		: Program which calculates x raised to the power of n, where x can be float number or integer and n can be positive or negative number
+# Example 1:
+
+# Input: x = 2.00000, n = 10
+# Output: 1024.00000
+# Example 2:
+
+# Input: x = 2.10000, n = 3
+# Output: 9.26100
+# Example 3:
+
+# Input: x = 2.00000, n = -2
+# Output: 0.25000
+# Explanation: 2^-2 = 1/(2^2) = 1/4 = 0.25
+
+#Class 
+class Solution:
+
```

Comment:
Also using docstring in class like this: to explain the brief task the program does, would be helpful.

```python
""" To output a given number to the power of n """

# if this is used for printing, see what happens.
# This is inbuilt dunder method.


print(function_name.__docs__)
```